config/ssl/localhost.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Don't commit private keys.

Reply:
If you need to generate a cert at build or runtime [here](https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/usr/bin/generate_miq_server_cert.sh) is an example of how we do it for manageiq